tools/src/main/resources/supportedExprs.csv

```diff
@@ -0,0 +1,643 @@
+Expression,Supported,SQL Func,Notes,Context,Params,BOOLEAN,BYTE,SHORT,INT,LONG,FLOAT,DOUBLE,DATE,TIMESTAMP,STRING,DECIMAL,NULL,BINARY,CALENDAR,ARRAY,MAP,STRUCT,UDT
+Abs,S,`abs`,None,project,input,NA,S,S,S,S,S,S,NA,NA,NA,S,NA,NA,NA,NA,NA,NA,NA
```

Comment:
did you load this up in Sark to make it was valid csv and no extra delimiters?  If not please check it

Reply:
Yes, I was able to read this and other csv files to a dataframe correctly.